src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/config.jelly

```diff
@@ -0,0 +1,3 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
```

Comment:
Does it appear in Web UI with such empty config? IIUC it may be confusing to user and, what is worse, cause undesired configuration overrides when submitting config pages

Reply:
In @dwnusbaum's review he tested it and said it worked fine:
https://github.com/jenkinsci/script-security-plugin/pull/233#pullrequestreview-179903699

Reply:
So can we close this as resolved?

Reply:
Things work fine with the file, but as far as I can tell everything works fine without it. Is there a reason it was added? If not, it should be deleted.

Reply:
I guess it is just there to satisfy `GlobalConfiguration.getGlobalConfigPage`. In that case maybe better to leave it, although could be good to add a comment here explaining why the empty file exists?